.buildkite/release-automation/pre_release.rayci.yml

```diff
@@ -16,6 +16,14 @@ steps:
     allow_dependency_failure: true
     commands:
       - bash .buildkite/release-automation/check-commit-hash.sh
+  
+  - label: "Build update version binary"
+    key: build-update-version-zip
+    instance_type: default
+    job_env: forge
+    commands:
+      - bazel build --build_python_zip --incompatible_use_python_toolchains=false --python_path=python //ci/ray_ci/automation:update_version
+      - cp bazel-bin/ci/ray_ci/automation/update_version.zip /artifact-mount/
```

Comment:
will you upload the binaries somewhere after this step? if so do you plan to automate that step too?

Reply:
I think the bot will just use buildkite API to download this from the pipeline.

Reply:
The binaries would not be uploaded anywhere else besides the build artifacts. And yes the automation bot would download this artifact using the Buildkite API (we already have a helper function just for that)